docs/content-crag/concepts/solids-pipelines/pipelines.mdx

```diff
@@ -1,196 +1,135 @@
-| <PyObject module="dagster" object="pipeline" displayText="@pipeline" /> | The decorator used to define a pipeline.                                                                                                                                                                         |
-| <PyObject module="dagster" object="PipelineDefinition" />               | Class for pipelines. You almost never want to use initialize this class directly. Instead, you should use the <PyObject object="pipeline" decorator /> which returns a <PyObject object="PipelineDefinition"  /> |
-| <PyObject module="dagster" object="PresetDefinition" />                 | Presets allow you to predefine pipeline configurations.                                                                                                                                                          |
+| Name                                                   | Description                                                                                                                                                                               |
+| ------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <PyObject module="dagster" object="graph" decorator /> | The decorator used to define a graph.                                                                                                                                                     |
+| <PyObject module="dagster" object="GraphDefinition" /> | Class for graphs. You almost never want to use initialize this class directly. Instead, you should use the <PyObject object="graph" decorator /> which returns an instance of this class. |
 
 ## Overview
 
-Solids are linked together into pipelines by defining the dependencies between their inputs and outputs. An important difference between Dagster and other workflow systems is that in Dagster, solids dependencies are expressed as data dependencies instead of just the order solids should execute in.
+Ops are linked together into graphs by defining the dependencies between their inputs and outputs. An important difference between Dagster and other workflow systems is that in Dagster, op dependencies are expressed as data dependencies, not just execution dependencies. Graphs can also contain other graphs.
 
 This difference enables Dagster to support richer modeling of dependencies. Instead of merely ensuring that the order of execution is correct, dependencies in Dagster provide a variety of compile and run-time checks.
 
-Dependencies are expressed in Pipelines using Dagster's function invocation DSL.
+Dependencies are expressed in graphs using Dagster's function invocation DSL.
```

Comment:
Nit: would be nice to replace DSL acronym with "domain-specific language". I was not sure what it was until googling 😅

Reply:
Good call